.github/workflows/daily-ci.yml

```diff
@@ -5,116 +5,92 @@ on:
   - cron: '0 4 * * *'
   workflow_dispatch:
 
-  ## COMMENTED OUT: only for testing CI action changes
+  # # COMMENTED OUT: only for testing CI action changes.
+  # # It only works for PRs to `main` branch from branches in the same gt4py repo.
   # pull_request:
   #   branches:
   #   - main
-  ## END
+  # # END
 
 jobs:
   daily-ci:
     strategy:
       matrix:
+        dependencies-factor: ["lowest-direct", "highest"]
```

Comment:
these are uv builtin names? maybe add a comment, to make it easy to look up their meaning

Reply:
Done.